packages/blocks/src/api/factory.js

```diff
@@ -537,11 +538,17 @@ export function switchToBlockType( blocks, name ) {
  * @return {Object} block.
  */
 export const getBlockFromExample = ( name, example ) => {
+	if ( ! getBlockType( name ) ) {
+		warning( 'Block example used an unknown block type: ' + name );
+		return false;
```

Comment:
The return type of the function would need to be updated in the doc block to be something like `@return {Object|false} block.`

I personally feel `undefined` would be a better return value than `false` to indicate the absence of a block, but I realise this is a nitpick 😄. If the return value were `undefined`, the doc block would then be `@return {?Object} block.` (I think that's how they're still declared).

Reply:
good idea with `undefined`. does the code/docs look better now?